app/main/views/suppliers.py

```diff
@@ -263,6 +264,56 @@ def edit_supplier_organisation_size():
     return render_template('suppliers/edit_supplier_organisation_size.html', form=form)
 
 
+@main.route('/vat-number/edit', methods=['GET', 'POST'])
+@login_required
+def edit_supplier_vat_number():
+    form = VatNumberForm()
+    try:
+        supplier = data_api_client.get_supplier(current_user.supplier_id)['suppliers']
+    except APIError as e:
+        abort(e.status_code)
+
+    if supplier.get("vatNumber"):
+        return (
+            render_template("suppliers/already_completed.html", completed_data_description="vat number"),
+            200 if request.method == 'GET' else 400
+        )
+
+    form_errors = None
+    if request.method == 'POST':
+        if form.validate_on_submit():
+            maybe_vat_number = form.vat_number.data if form.vat_registered.data == 'Yes' else 'Not VAT registered'
```

Comment:
Isn't `maybe_vat_number` an odd variable name? Maybe we could rename it? Something like `vat_number_info` potentially? What do you think?

Reply:
Yeah it's a bit weird. I know we've used similar before though in the search-api. Happy to change it here.